nupic/torch/functions/k_winners.py

```diff
@@ -21,229 +21,187 @@
 import torch
 
 
-class KWinners(torch.autograd.Function):
-    """A simple K-winner take all autograd function for creating layers with
-    sparse output.
-
-    .. note::
-        Code adapted from this excellent tutorial:
-        https://github.com/jcjohnson/pytorch-examples
+def kwinners(x, duty_cycles, k, boost_strength, break_ties=True, relu=False,
+             inplace=False):
```

Comment:
Maybe we should default to `break_ties=False` assuming it is unlikely it will impact the accuracy on most common use cases (float32)